zk-states/src/types.ts

```diff
@@ -128,8 +128,20 @@ interface ProofSuccessUpdate {
   updateType: "proofSuccess";
   callId: string;
 }
+export type initializationProgress =
+  | "initializing"
+  | "compiling program"
+  | "compiling contract"
+  | "creating initial proof"
+  | "initialization done";
+
+interface inititializationProgressUpdate {
```

Comment:
```suggestion
interface InititializationProgressUpdate {
```
types are never lower case